### PR TITLE
feat: removed indiscriminate * imports for explicit ones

### PR DESCRIPTION
**Pull request by LeonFLK**
## fixes KILTProtocol/ticket#857

Has no logic changes, just removes indiscriminate `* as sdk` imports in favor for explicit ones that are needed.

## Checklist:

- [x] I have verified that the code works
- [x] I have verified that the code is easy to understand
  - [ ] If not, I have left a well-balanced amount of inline comments
- [x] I have [left the code in a better state](https://deviq.com/boy-scout-rule/)
- [x] I have documented the changes (where applicable)

### CODE REVIEW

**Review by rflechtner — CHANGES_REQUESTED** [review]
Just a typo, but it doesn't build like this. LG otherwise

File: src/components/SelectAttestedClaim/SelectAttestedClaim.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ import { getCtypePropertyTitle } from '../../utils/CtypeUtils'
 
 type Props = {
   claimEntry: Claims.Entry
-  cTypeHash?: sdk.ICType['hash']
+  cTypeHash?: SDKICTypeu['hash']
```
Comment (LeonFLK) [inline src/components/SelectAttestedClaim/SelectAttestedClaim.tsx:17]:
```suggestion
  cTypeHash?: SDKICType['hash']
```

[the remark this answers is not in the document]
Comment (LeonFLK) [reply src/components/SelectAttestedClaim/SelectAttestedClaim.tsx]: WEEOO-WEEEOO
Danke, aber da hättest du auch ne suggestion commiten können.

[the remark this answers is not in the document]
Comment (rflechtner) [reply src/components/SelectAttestedClaim/SelectAttestedClaim.tsx]: das wär ja langweilig!

**Pull request merged**